client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -53,7 +53,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I install plugins?' ) }
 				answer={ translate(
-					'Yes! With the WordPress.com Business plan you can search for and install external plugins.' +
+					'Yes! With the WordPress.com Business or eCommerce plan you can search for and install external plugins.' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Yes! With the WordPress.com Business plan you can search for and install external plugins. All plans already come with a custom set of plugins tailored just for them. {{a}}Find out more about plugins{{/a}}.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).